### ISSUE TITLE
I'm trying to train a classifier model using darknet, can I use any cfg file in the darknet/cfg folder?

### ISSUE THREAD

**Issue by muyue1238** [pjreddie/darknet#1208]
I'm trying to train a classifier model using darknet, and I followed this [tutorial](https://pjreddie.com/darknet/imagenet/), and my training command is like this:
`./darknet classifier train cfg/my_classifier.data cfg/darknet19.cfg `
the 'my_classifier.data' file has been modified according to my dataset.
Anyhow, I'm a little confused about the darknet19.cfg. My question is that there are dozen of cfg files the darknet/cfg folder, can I just choose one cfg file and use it to train a classifier model, or I can just use certain cfg file to train it?
As far as I know, the darknet19_448.cfg can be used during detector-training process, and it seems there's no big difference between darknet19.cfg and darknet19_448.cfg.
Could someone please explain for me.
FYI, I've trained my classifier model for about 10000 times, and it worked out not very well, it may be my dataset's problem, but I want to be sure, that's why I want to know if I can use another cfg file to achieve better accuracy.

**Comment by han-so1omon** [comment]
Just trying simple `detect` action following MMaxV's suggestions. Getting `Segmentation fault (core dumped)`. Also using 2 classes.

**Comment by YunYang1994** [comment]
https://github.com/YunYang1994/tensorflow-yolov3 hope it helps you

**Comment by FranciscoReveriano** [comment]
Does anyone have any advice to convert the weights pre-trained in YOLOv3 into classification weights?
Basically no YOLO just classification?

**Comment by water5tyle** [comment]
@FranciscoReveriano, you should use darknet53 model. It's classifier that is used in yolov3.

**Comment by zaimpower** [comment]
> @FranciscoReveriano, you should use darknet53 model. It's classifier that is used in yolov3.

Hi can you briefly explain that because I am working on only classifier not YOLOv3

**Comment by water5tyle** [comment]
@zaimpower You can read about this on original repo [https://pjreddie.com/darknet/imagenet/]

**Comment by zaimpower** [comment]
> @zaimpower You can read about this on original repo (https://pjreddie.com/darknet/imagenet/)

Yes i have try using it, but that only uses one image. How about my own custom dataset.  Will you help :(

**Comment by water5tyle** [comment]
@zaimpower you can find classification examples on python here https://github.com/AlexeyAB/darknet
